.github/workflows/canada_staging.yml

```diff
@@ -46,7 +46,7 @@ jobs:
         with:
           args: --acl public-read --follow-symlinks --exclude '*' --include '*.js' --content-type 'application/javascript; charset=utf-8'
         env:
-          AWS_S3_BUCKET: tl-public-chat-ca-staging
+          AWS_S3_BUCKET: tl-public-chat-ca-stg
```

Comment:
Is it possible that this change is against the naming convention on the buckets?

Reply:
This is discussed here: https://tech-matters.slack.com/archives/CV4QM23UJ/p1644485405774699 - does that seem reasonable or do you have concerns?